src/BusinessLogic/Resources/countries/en.json

```diff
@@ -309,7 +310,8 @@
     "submit": "Register",
     "chooseYourCountry": "Choose your country",
     "searchCountry": "Search country",
-    "invalidDeliveryVolume": "Field is not a valid delivery volume."
+    "invalidDeliveryVolume": "Field is not a valid delivery volume.",
+    "passwordRestriction": "Your password must be at least 12 characters long and include at least one lowercase letter, one uppercase letter, one number, and one symbol"
```

Comment:
Suggestion: replace `12` as fixed number for a parameter when using this expression